podpac/core/interpolation/interpolators.py

```diff
@@ -368,7 +368,7 @@ def interpolate(self, udims, source_coordinates, source_data, eval_coordinates,
             if order == "lat_lon":
                 pts = pts[:, ::-1]
             pts = KDTree(pts)
-            lon, lat = np.meshgrid(eval_coordinates.coords["lon"], eval_coordinates.coords["lat"])
+            lon, lat = np.meshgrid(eval_coordinates["lon"].coordinates, eval_coordinates["lat"].coordinates)
```

Comment:
I wonder how `git` will handle the fact that I deleted this file in the other branch... should be interesting merge. Make a note that this change is retained after the merge.

Reply:
As suspected, git had no idea. Fixed.